apps/e2e-tests/src/process-compose/helpers.ts

```diff
@@ -64,3 +67,27 @@ export function logTestEnvironmentInfo(
     `${status} time: ${time.toDateString()} ${time.toTimeString()}`,
   );
 }
+
+export const rgSearchPattern = ({
+  caseInsensitive = true,
+  file,
+  flags = [],
+  pattern,
+}: {
+  caseInsensitive?: boolean;
+  file: string;
+  flags?: Array<string>;
+  pattern: string;
+}) => {
```

Comment:
Add return type to make the behaviour clear:

* If goal of this function is to only tell if a match has been found, the success type should be `boolean`
* If it you need the matches (say get all logs with "result from network") then perhaps it should return:
  ```ts
  type RipGrepMatch = {
    file: string;
    lineNumber: number;
    match: string; // Consider regex match or the whole line
  }[];
  ```

Perhaps there should be one generic function (that takes a list of flags) and some helpers that either return boolean or list of results on top.